spec/acceptance/class_spec.rb

```diff
@@ -45,16 +45,29 @@ class { 'apache':
         include epel
         class { 'apache::mod::wsgi': wsgi_socket_prefix => '/var/run/wsgi' }
       } else {
-        class { 'apache::mod::wsgi': }
+        class { 'apache::mod::wsgi':
+          package_name => "libapache2-mod-wsgi-py3",
+          mod_path     => "/usr/lib/apache2/modules/mod_wsgi.so",
+        }
       }
 
+      # Configure PuppetDB
+      class { 'puppetdb': disable_ssl => true, }
+
       # Configure Puppetboard
-      class { 'puppetboard': }
+      class { 'puppetboard':
+        manage_virtualenv  => true,
+        virtualenv_version => '3',
```

Comment:
I really think 3 should be the default, but that maybe should be changed in the future/another PR.